dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       },
       {
         "value": "640GC",
-        "label": "FREMONT VA CLINIC"
+        "label": "Fremont VA Clinic"
       },
       {
         "value": "654GD",
-        "label": "DIAMOND VIEW VA CLINIC"
+        "label": "Diamond View VA Clinic"
       },
       {
         "value": "570GC",
-        "label": "OAKHURST VA CLINIC"
+        "label": "Oakhurst VA Clinic"
       },
       {
         "value": "662GG",
-        "label": "CLEARLAKE VA CLINIC"
+        "label": "Clearlake VA Clinic"
       },
       {
         "value": "612GI",
-        "label": "YUBA CITY CBOC"
+        "label": "Yuba City VA Clinic"
       },
       {
         "value": "691A4",
-        "label": "SEPULVEDA OUTPATIENT & NURSING HOME CARE"
+        "label": "Sepulveda VA Medical Center"
       },
       {
         "value": "691GB",
-        "label": "SANTA BARBARA CBOC"
+        "label": "Santa Barbara VA Clinic"
       },
       {
         "value": "691GC",
-        "label": "GARDENA CBOC"
+        "label": "Gardena VA Clinic"
       },
       {
         "value": "691GD",
-        "label": "BAKERSFIELD CBOC"
+        "label": "Bakersfield VA Clinic"
       },
       {
         "value": "612GJ",
-        "label": "YREKA VA CLINIC"
+        "label": "Yreka VA Clinic"
       },
       {
         "value": "605BZ",
-        "label": "LOMA LINDA VA CLINIC"
+        "label": "Loma Linda VA Clinic"
       },
       {
         "value": "612QD",
-        "label": "HOWE ROAD VA CLINIC"
+        "label": "Howe Road VA Clinic"
       },
       {
         "value": "600",
-        "label": "VA LONG BEACH HEALTHCARE SYSTEM"
+        "label": "Tibor Rubin VA Medical Center"
       },
       {
         "value": "605",
-        "label": "LOMA LINDA HCS"
+        "label": "Jerry L. Pettis Memorial Veterans' Hospital"
```

Comment:
Let's confirm this name change.

Reply:
Flagging this label since it goes from an entire health care system to a single facility.